app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def render_title
+    content_for?(:title) ? "#{content_for(:title)} | RG Portal" : 'RG Portal'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.